src/Redis/Tests/Caching/RedisCacheClientTests.cs

```diff
@@ -68,8 +69,19 @@ public class RedisCacheClientTests : CacheClientTestsBase {
         }
 
         [Fact]
-        public override Task CanManageSets() {
-            return base.CanManageSets();
+        public override async Task CanManageSets() {
+            await base.CanManageSets();
+
+            var cache = GetCacheClient();
+            if (cache == null)
+                return;
+
+            using (cache) {
+                await Assert.ThrowsAsync<StackExchange.Redis.RedisServerException>(async () => {
```

Comment:
I believe we should be checking this in all the cache clients the same way. And verify the key value hasn't been changed or set to null (do we agree on this @ejsmith).